### PR TITLE
apps/wallet/{sign_identity, ecdh}: show protocol as part of the header

### PR DESCRIPTION
**Pull request by romanz**
This PR should make signature/decryption dialogs a bit nicer (compared to https://github.com/trezor/trezor-core/pull/158):
screenshot_2018-03-14_11-15-06screenshot_2018-03-14_11-15-20
What do you think?

### CODE REVIEW

File: src/apps/wallet/ecdh.py [hunk 1]
```diff
@@ -4,7 +4,8 @@
 from ustruct import pack, unpack
 from trezor.utils import chunks
 from apps.common.confirm import require_confirm
-from apps.wallet.sign_identity import serialize_identity
+from apps.wallet.sign_identity import (serialize_identity,
```
Comment (prusnak) [inline src/apps/wallet/ecdh.py]: Style nitpick: please don't use braces and keep import on one line.

File: src/apps/wallet/ecdh.py [hunk 2]
```diff
@@ -28,8 +29,10 @@
 
 
 async def require_confirm_ecdh_session_key(ctx, identity):
-    lines = chunks(identity, 18)
-    content = Text('Decrypt', ui.ICON_DEFAULT, ui.MONO, *lines, max_lines=5)
+    lines = chunks(serialize_identity_without_proto(identity), 18)
+    header = 'Decrypt {}'.format(identity.proto.upper()
```
Comment (prusnak) [inline src/apps/wallet/ecdh.py]: Style nitpick: please use modulo formatting for consistency.

File: src/apps/wallet/sign_identity.py [hunk 3]
```diff
@@ -44,10 +45,16 @@
...
-                   challenge_visual,
-                   ui.MONO, *lines, max_lines=5)
+    lines = []
+    if challenge_visual:
+        lines.append(challenge_visual)
+
+    lines.append(ui.MONO)
+    lines.extend(chunks(serialize_identity_without_proto(identity), 18))
+
+    header = 'Sign {}'.format(identity.proto.upper()
```
Comment (prusnak) [inline src/apps/wallet/sign_identity.py]: Style nitpick: please use modulo formatting for consistency.

File: src/apps/wallet/ecdh.py (hunk 2 above)
Comment (romanz) [reply src/apps/wallet/ecdh.py]: Done.

File: src/apps/wallet/ecdh.py (hunk 1 above)
Comment (romanz) [reply src/apps/wallet/ecdh.py]: Done.

**Comment by prusnak** [comment]
Overally, I really like the change.

**Comment by prusnak** [comment]
Thanks!

**Comment by romanz** [comment]
My pleasure :)

**Pull request merged**